rustls/src/msgs/handshake.rs

```diff
@@ -1256,7 +1256,26 @@ impl ServerHelloPayload {
     }
 }
 
-pub(crate) type CertificatePayload = Vec<CertificateDer<'static>>;
+#[derive(Clone, Default, Debug)]
+pub struct CertificateChain(pub Vec<CertificateDer<'static>>);
+
+impl core::ops::Deref for CertificateChain {
```

Comment:
Please import `Deref` at the top of the module and move this `impl` below the `Codec` impl.